tutorial03/my-model-dir/table/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
```

Comment:
The tutorial instructs the user to make a yaml file. Now there is both a yaml file for a table and an sdf file. I dont know which one ED will take here but this is definitly bad practice. 
The tutorial also does not mention how to create an sdf file. If you want to update the models you should also update the tutorial itself.